recipes/configure.rb

```diff
@@ -14,7 +14,7 @@
     mode "0660"
     group deploy[:group]
     owner deploy[:user]
-    variables(:database => deploy[:database], :environment => deploy[:rails_env])
+    variables(:database => deploy[:database], :environment => deploy[:rails_env], :deploy => deploy)
```

Comment:
Yeah, sorry for the confusion. I pulled both the open PRs into my local and there is a merge conflict when pulling both in. More of a warning for you all.